server/app/views/index.scala.html

```diff
@@ -1,13 +1,13 @@
 @import UserController._
 @import views.html.templates.mainFlashToUser
 
-@()(implicit flash: Flash, session: Session, msg: Messages)
+@(tasks: Iterable[Task])(implicit flash: Flash, session: Session, msg: Messages)
 @mainFlashToUser(msg("home")) {
-    <div><a href="@routes.TaskSolver.getTask()">@msg("task")</a></div>
-    <div><a href="@routes.NewTask.getAddTask()">@msg("addTask")</a></div>
-    @session.get(username).fold {
-        <div><a href="@routes.UserController.getRegister()">@msg("register")</a></div>
-    } { _ =>
-        <div><a href="@routes.Application.logout()">@msg("logout")</a></div>
+    <br/>
+    <h1>Latest Challenges</h1>
```

Comment:
yeah, styles should be changed in the future to some consistent design accross all pages of DevGym.